Libraries/Alert/NativeAlertManager.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+type Button = {|
+  text?: string,
+  onPress?: () => void,
+  style?: string,
+|};
+
+type Args = {|
```

Comment:
Ditto

Reply:
Should `Args` be located at Alert.js?

Reply:
I'm not sure, but I would keep it in this file